src/PointwiseFunctions/GeneralRelativity/Christoffel.hpp

```diff
@@ -38,4 +45,68 @@ template <size_t SpatialDim, typename Frame, IndexType Index, typename DataType>
 tnsr::abb<DataType, SpatialDim, Frame, Index> christoffel_first_kind(
     const tnsr::abb<DataType, SpatialDim, Frame, Index>& d_metric) noexcept;
 // @}
-} // namespace gr
+namespace Tags {
+/// Compute item for spatial Christoffel symbols of the first kind
+/// \f$\Gamma_{ijk}\f$ computed from the first derivative of the
+/// spatial metric.
+///
+/// Can be retrieved using `gr::Tags::SpatialChristoffelFirstKind`
+template <size_t SpatialDim, typename Frame, typename DataType>
+struct SpatialChristoffelFirstKindCompute
+    : SpatialChristoffelFirstKind<SpatialDim, Frame, DataType>,
+      db::ComputeTag {
+  using argument_tags = tmpl::list<
+      ::Tags::deriv<gr::Tags::SpatialMetric<SpatialDim, Frame, DataType>,
+                    tmpl::size_t<SpatialDim>, Frame>>;
+  static constexpr tnsr::ijj<DataType, SpatialDim, Frame> (*function)(
+      const tnsr::ijj<DataType, SpatialDim, Frame>&) =
+      &christoffel_first_kind<SpatialDim, Frame, IndexType::Spatial, DataType>;
+  using base = SpatialChristoffelFirstKind<SpatialDim, Frame, DataType>;
+  using type = typename base::type;
+};
+
+/// Compute item for spatial Christoffel symbols of the second kind
+/// \f$\Gamma^i_{jk}\f$ computed from the Christoffel symbols of the
+/// first kind and the inverse spatial metric.
+///
+/// Can be retrieved using `gr::Tags::SpatialChristoffelSecondKind`
+template <size_t SpatialDim, typename Frame, typename DataType>
+struct SpatialChristoffelSecondKindCompute
+    : SpatialChristoffelSecondKind<SpatialDim, Frame, DataType>,
+      db::ComputeTag {
+  using argument_tags =
+      tmpl::list<SpatialChristoffelFirstKind<SpatialDim, Frame, DataType>,
+                 InverseSpatialMetric<SpatialDim, Frame, DataType>>;
+  static constexpr tnsr::Ijj<DataType, SpatialDim, Frame> (*function)(
+      const tnsr::ijj<DataType, SpatialDim, Frame>&,
+      const tnsr::II<DataType, SpatialDim, Frame>&) =
+      &raise_or_lower_first_index<DataType,
+                                  SpatialIndex<SpatialDim, UpLo::Lo, Frame>,
+                                  SpatialIndex<SpatialDim, UpLo::Lo, Frame>>;
+  using base = SpatialChristoffelSecondKind<SpatialDim, Frame, DataType>;
```

Comment:
`base` and `type` are not part of the compute item interface. Please remove these.

Reply:
I'll remove `type`...

... but `base` is needed when putting compute items on domain interfaces via `InterfaceComputeItem` (which inherits from `Interface<, Tag>`, which in turn inherits from `Interface<,Tag::base>`). While these compute items are not yet needed on domain interfaces, they might be in future.